Sources/KSCrashFilters/include/KSCrashReportFilterJSON.h

```diff
@@ -39,7 +39,8 @@ NS_ASSUME_NONNULL_BEGIN
 NS_SWIFT_NAME(CrashReportFilterJSONEncode)
 @interface KSCrashReportFilterJSONEncode : NSObject <KSCrashReportFilter>
 
-+ (instancetype)filterWithOptions:(KSJSONEncodeOption)options;
+- (instancetype)init NS_UNAVAILABLE;
++ (instancetype)new NS_UNAVAILABLE;
```

Comment:
Do we have to provide options?

Reply:
I thought it would be best to make the options explicit, as it's not possible for the user to change them once the class has been initialised.